src/librustc/hir/svh.rs

```diff
@@ -74,14 +48,16 @@ impl Svh {
             b as char
         }
     }
+}
 
-    pub fn as_str<'a>(&'a self) -> &'a str {
-        &self.hash
+impl Hash for Svh {
+    fn hash<H>(&self, state: &mut H) where H: Hasher {
+        self.hash.to_le().hash(state);
```

Comment:
Does this really do what it seems to be doing? I always find myself wishing that `to_le()` would return a `[u8; BYTE_COUNT]` to make things clearer.